src/components/__stories__/simple_nav.tsx

```diff
@@ -0,0 +1,23 @@
+import { action, storiesOf } from "@kadira/storybook"
+import * as React from "react"
+
+import styled from "styled-components"
+import SimpleNav from "../simple_nav"
+import TextLink from "../text_link"
+
+import * as fonts from "../../assets/fonts"
+
+const LinkContainer = styled.span`
+  position: relative;
+  top: -8px;
+  font-family: ${ fonts.primary.fontFamily };
+`
+
+storiesOf("Simple Nav", SimpleNav)
+  .add("Simple Nav Bar", () => (
+    <SimpleNav>
+      <LinkContainer>
+        <TextLink href="https://www.artsy.net">Back To Artsy</TextLink>
```

Comment:
@xtina-starr I think the styling for the link should be encapsulated in a component. In terms of the `Link` component, I agree that it's a similar case to the Title Component. 

Also I would suggest naming the component `Nav`, we can decide on whether it's simple or complex based on the children components.

So the component could look like this.

```javascript
const Link = styled.a`
  position: relative;
  ...
`

<Nav>
  <Link href="https://www.artsy.net">Back To Artsy</Link>
</Nav>
```